python/polycap.pyx

```diff
@@ -544,22 +1023,104 @@ cdef class Photon:
 
         return weights_np
 
+    @property
+    def extleaks(self):
```

Comment:
Denk dat het beter is dat de method names meer overeenkomen met hun C- counterparts:

`extleak_data` ipv `extleaks`?